### ISSUE TITLE
Unable to set the marketing option as already checked

### ISSUE THREAD

**Issue by theoldcounty** [Palmabit-IT/react-cookie-law#26]
Not seen any params to set checkboxes as already set to true?

**Comment by gerardparareda** [comment]
```
window.requestAnimationFrame(() => {
      document.getElementById("check-statistics-cookies").checked = false;
      document.getElementById("check-preferences-cookies").checked = false;
    })
```
You can access the checkbox directly via id. Not a good solution in the slightest but it does the work.

**Comment by asarzina** [comment]
Resolved with version 0.6.2. Now you can use the props `marketingDefaultChecked`
